docs(examples): Add cms-batch task (#437)
piskoviste/pisek: examples/cms-batch/statement.md

# Building sandcastles

> Timmy is at the beach and he wants to build sandcastles for his little kingdom.
> He has already dig up $K$ piles of sand and wants to use **all** of them to build $N$ sandcastles. 
> Tell him how many piles of sand should end in each castle. 

Reviewer: ```suggestion
Timmy is at the beach and he wants to build sandcastles for his little kingdom.
He has already dug up $K$ piles of sand and wants to use **all** of them to build $N$ sandcastles. 
Tell him how many piles of sand should end up in each castle. 
```

## Input format

> On the only line you will get two positive integers — $N$ and $K$ respectively.

Reviewer: ```suggestion
On the only line of input you will get two positive integers — $N$ and $K$.
```

## Output format

> On the only line output $N$ positive integers — size of each sandcastle. 
> The size of the sandcastles should sum up to $K$.

Reviewer: ```suggestion
Output a single line with $N$ positive integers — the size of each sandcastle. 
The sizes of the sandcastles should sum up to $K$.
```
